seqr/views/apis/staff_api_tests.py

```diff
@@ -375,6 +376,80 @@
     'entity:discovery_id': u'HG00731', 'Chrom-1': u'1',
     'inheritance_description-1': 'de novo'}
 
+EXPECTED_SAMPLE_METADATA_ROW = {
+    "project_guid": "R0001_1kg",
+    "num_saved_variants": 1,
+    "dbgap_submission": "Yes",
+    "solve_state": "Tier 1",
+    "sample_id": "NA19675",
+    "Gene_Class-1": "Known",
+    "sample_provider": "Hildebrandt",
+    "inheritance_description-1": "de novo",
+    "hpo_present": "HP:0001631 (Defect in the atrial septum)|HP:0002011 (Morphological abnormality of the central nervous system)|HP:0001636 (Tetralogy of Fallot)",
+    "novel_mendelian_gene-1": "Y",
+    "hgvsc-1": "c.375_377delTCT",
+    "date_data_generation": "2017-02-05",
+    "dbgap_subject_id": "dbgap_subject_id_1",
+    "Zygosity-1": "Heterozygous",
+    "dbgap_study_id": "dbgap_stady_id_1",
+    "Ref-1": "GAGA",
+    "multiple_datasets": "No",
+    "ancestry_detail": "",
+    "maternal_id": "NA19679",
+    "paternal_id": "NA19678",
+    "hgvsp-1": "p.Leu126del",
+    "entity:family_id": "NA19675_1",
+    "entity:discovery_id": "NA19675_1",
+    "project_id": u"1kg project n\xe5me with uni\xe7\xf8de",
+    "Pos-1": "3343353",
+    "data_type": "WES",
+    "family_guid": "F000001_1",
+    "onset_category": "Adult onset",
+    "hpo_absent": "HP:0011675 (Arrhythmia)|HP:0001674 (Complete atrioventricular canal defect)|HP:0001508 (Failure to thrive)",
+    "Transcript-1": "ENST00000258436",
+    "dbgap_sample_id": "SM-A4GQ4",
+    "ancestry": "",
+    "phenotype_group": "",
+    "sex": "Male",
+    "entity:subject_id": "NA19675_1",
+    "entity:sample_id": "NA19675_1",
+    "Chrom-1": "21",
+    "Alt-1": "G",
+    "Gene-1": "RP11-206L10.5",
+    "pmid_id": "",
+    "consanguinity": "Present",
+    "phenotype_description": "",
+    "affected_status": "Affected",
+    "family_id": "1",
+    "MME": "Y",
+    "subject_id": "NA19675_1"
+  }
+
+SAMPLE_QC_DATA = [
+    'seqr_id	data_type	filter_flags	qc_platform	qc_pop	qc_metrics_filters\n',
+    '03133B_2	WES	[]	Standard Germline Exome v5	nfe	[]\n',
+    'NA19675	WES	[]	Unknown	nfe	["n_deletion","n_insertion","n_snp"]\n',
+    'NA19678	WES	[]	Unknown	amr	["n_deletion","n_insertion","n_snp"]\n',
+    'HG00732	WES	["coverage"]	Standard Exome Sequencing v4	nfe	["r_insertion_deletion"]\n',
```

Comment:
You should also keep in the `HS_PCT_TARGET_BASES_20X` column and the `AL_PCT_CHIMERAS`, make sure you have at least one row where there is a `coverage` filter in filter_flags (which you have) and at least one row with `chimera` (which you don't). Then you should be testing the filter_flags dict on the Individual has the value from those columns

Reply:
Updated